README.md

```diff
@@ -24,33 +19,23 @@
 
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
+### Clone the repository
 
```

Comment:
Great job including the getting started instructions 👍🏽
- [ ]  it would be more useful if you include commands as part of the instructions for the instructions that use commands for example the command for cloning the repository.